install/kubernetes/helm/istio/charts/certmanager/templates/crds.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
     "helm.sh/hook": crd-install
   labels:
     app: certmanager
+    chart: certmanager
```

Comment:
is it possible for this to use variable like galley at https://github.com/istio/istio/pull/8793/files#diff-7d741f98adfe8d65d972d95dd70fb1fbR7

Reply:
@gyliu513 If we replace it with variables, we may encounter issues when creating CRDs with `kubectl apply -f` because they need to be rendered with Tiller.